package.json

```diff
@@ -154,6 +174,35 @@
           "command": "extension.runQuery",
           "when": "editorLangId == sql"
         }
+      ],
+      "view/title": [
+        {
+          "command": "extension.addObjectExplorer",
+          "title": "%extension.addObjectExplorer%",
+          "group": "navigation"
+        }
+      ],
+      "view/item/context": [
+        {
+          "command": "extension.objectExplorerNewQuery",
+          "when": "viewItem == Server"
+        },
+        {
+          "command": "extension.objectExplorerNewQuery",
+          "when": "viewItem == Database"
+        },
+        {
+          "command": "extension.objectExplorerNewQuery",
```

Comment:
if we could simply add "SELECT TOP 1000" here that would be awesome.  That's one of the top use-cases users want to complete when viewing a table in OE.